### PR TITLE
Feat/updated homepage

### LINKED ISSUE #37: feat/updated-homepage
## Value statement - a requirement, not a solution
As a visitor
I need to understand what the app offers
so that I can decide if I want to use it.

## Description (Text, Scribble, Wireframe, Design)
With the addition of the journal function (#36) the homepage needs to offer access to both questionnaire and journal sections. Both parts need to be introduced and have some sort of link component.

Home_New

## Acceptance criteria
- [x] the user can start the motive selection
- [x] the user can got to the journal page

## Tasks
- [x] update Homepage component
- [x] write/update tests
- [x] update Storybook (if necessary)

## Size
medium

### PR DESCRIPTION
**Pull request by karstenlenz**
Please review my code.

closes #37

### CODE REVIEW

File: src/Homepage/Homepage.js [hunk 1]
```diff
@@ -1,44 +1,98 @@
...
+          <li>
+            Entwickeln Sie ein besseres Verständnis für sich selbst in
+            zwischenmenschlichen Beziehungen.
+          </li>
+          <li>
+            Profitieren Sie von speziell auf Sie zugeschnittenen Alltagstipps.
+          </li>
+        </BulletList>
+        <Link to="/motives/intro">
+          <Button>Test starten</Button>
```
Comment (maximiliangustmann) [inline src/Homepage/Homepage.js:57]: I would remove the Button and style the Link instead

File: src/Homepage/Homepage.js [hunk 2]
```diff
@@ -1,44 +1,98 @@
...
+          <li>
+            Übernehmen Sie die Kontrolle: Selbstbeobachtung ermöglicht
+            Veränderung.
+          </li>
+          <li>
+            Nutzen Sie das Tagebuch als Gesprächsstütze für Therapie-Termine.
+          </li>
+        </BulletList>
+        <Link to="/journal">
+          <Button>Tagebuch öffnen</Button>
```
Comment (maximiliangustmann) [inline src/Homepage/Homepage.js:78]: Same here

**Review by maximiliangustmann — COMMENTED** [review]
Looks good to me :-)

File: src/Homepage/Homepage.js (hunk 1 above)
Comment (karstenlenz) [reply src/Homepage/Homepage.js:57]: I thought about that too, but then I would have duplicate styles for the "button"-button and the "link"-Button. So I think this is cleaner, even if I have to wrap the button in the Link-component.

**Comment by karstenlenz** [comment]
> Looks good to me :-)

Thank you very much for the code review!

**Pull request merged**